src/js/components/ExpandingWithCheckboxTable.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import CheckboxTable from './CheckboxTable';
+import ExpandingTable from './ExpandingTable';
+
+class ExpandingWithCheckboxTable extends React.Component {
+  getColGroup() {
+    return (
+      <colgroup>
+        <col style={{width: '40px'}} />
+        <col />
+        <col style={{width: '120px'}} className="hidden-mini" />
+        <col style={{width: '160px'}} className="hidden-mini" />
+        <col style={{width: '160px'}} className="hidden-mini" />
```

Comment:
Yeah. My initial plans were more ambitious in terms of what I wanted to achieve with this. But without any further refactorings on the CheckboxTable and ExpandingTable, this "composite" component doesn't make much sense anymore. I will move this to the JobRunHistoryTable itself to keep things simple.